src/widgets/mobile-view/ui/mobile-view.tsx

```diff
@@ -1,59 +1,54 @@
+import classNames from 'classnames/bind';
 import { Link } from 'react-router-dom';
 import { ROUTES } from '@/app/const';
 import { Logo } from '@/shared/ui/logo';
 import { SchoolMenu } from '@/widgets/school-menu';
 
-import './mobile-view.scss';
+import styles from './mobile-view.module.scss';
+
+const cx = classNames.bind(styles);
 
 interface MobileViewProps {
   type: 'header' | 'footer';
 }
 
 const Divider = ({ type }: MobileViewProps) => (
-  <div className={`divider ${type === 'header' ? 'dark' : 'light'}`} />
+  <div className={cx('divider', type === 'header' ? 'dark' : 'light')} />
```

Comment:
```suggestion
  <hr className={cx('divider', 'light', { dark: type === 'header' })} />
```

Reply:
Please note, that the `hr` by default has `border-top` style if you apply the suggested changes don't forget to reset it

Reply:
changed tag to hr, but not sure about conditionally adding one class, as for me readability suffers and not sure that it's good idea to combine classes with overwriting style properties instead of jsut replacing class.

Reply:
for me the whole point of adding `classNames` was to get rid of annoying ternary operator

i'm going to approve the pr but leave the conversation unresolved, maybe someone else will have an opinion about that

Reply:
I decided to pass color directly to divider as it's already calculated from type in MobileView component.  No more ternary in divider :)
please check commit 4e33a6dfae89232823f78d20b31b3985fbe0dc3c

Reply:
🔥🔥🔥